IP_POLICY_GUIDANCE.md

```diff
@@ -63,8 +63,9 @@ Each project must adopt either the [Developer Certificate of Origin (DCO)][DCO]
 or a Contributor License Agreement (CLA).
 
 The Board has pre-approved an [Individual CLA (ICLA)][ICLA]
-and a [Corporate CLA (CCLA)][CCLA]
-which projects can adopt without any further review.
+and a [Corporate CLA (CCLA)][CCLA].
+Projects can adopt either the ICLA by itself or
+both the ICLA and CCLA without any further review.
```

Comment:
Do we really need to define and use the ICLA and CCLA acronyms, rather than just referring to them as the "Individual CLA" and "Corporate CLA"? As far as I can tell, this is the only place they're used, besides CPC meeting minutes.

Reply:
Good point. Done.